### ISSUE TITLE
Theme appearing too dark

### ISSUE THREAD

**Issue by benmaughan** [ksjogo/labburn-theme#3]
Hi, I usually use zenburn so wanted to try out your version. I enabled it using

```
  (use-package labburn-theme
    :config
    (load-theme 'labburn t))

```

but it looks nothing like the screenshot - it is very dark with most text being hard to even see (see screenshot).

I am using GNU Emacs 24.5.1 (x86_64-apple-darwin14.4.0, NS apple-appkit-1348.17) from Macports on OS X

Thanks!

![screen shot 2016-05-18 at 12 43 35](https://cloud.githubusercontent.com/assets/10402065/15357370/47c08d60-1cf6-11e6-8fb3-fb8e05ec1930.png)

**Comment by ksjogo** [comment]
That indeed seems way to dark. Will look into it.

**Comment by ksjogo** [comment]
Could you try starting emacs with -Q and then evaling the buffer? It looks like the theme is not overwritting some settings from the previously active theme. No idea what causes that though.

**Comment by ericho** [comment]
Same issue here, I did the -Q and then eval-buffer in the labburn-theme.el, the theme went dark again. 
I did the same thing from the terminal with emacs -nw and the theme was also dark. 

I'm using emacs 24.5.1. 

Is this theme requiring some package? Maybe something is missing in the installation. 
Also, it's possible to get the output from eval-buffer? I don't know, some debug info that could be more helpful to root cause the issue. 

Thanks
